### ISSUE TITLE
scala 2.13.0 release

### ISSUE THREAD

**Issue by pjfanning** [NewMotion/akka-rabbitmq#54]
Would it be possible to do a scala 2.13.0 release?

I tried to build akka-rabbitmq with scala 2.13.0 but https://github.com/NewMotion/sbt-build-seed/blob/master/BasicPlugin.scala#L37 causes a problem.

```
[error] bad option: '-Ywarn-unused-import'
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 14 s, completed 20-Jun-2019 00:02:54
```

https://github.com/pjfanning/akka-rabbitmq/commit/d0c8d83db4e5cdfa8eb32de4009facd0dde94010

**Comment by pjfanning** [comment]
I pushed a forked release to support scala 2.13.0. https://github.com/pjfanning/akka-rabbitmq

**Comment by kiequoo** [comment]
Published `5.1.1` cross-built for Scala 2.11, 2.12 and 2.13
